owncloudApp/src/test/java/com/owncloud/android/NetworkBoundResourceTest.kt

```diff
@@ -120,12 +126,16 @@ class NetworkBoundResourceTest {
 
         val networkResult: ArrayList<RemoteShare> = arrayListOf()
 
-        var networkBoundResource =
+        val networkBoundResource =
             object : NetworkBoundResource<List<OCShare>, ShareParserResult>(InstantAppExecutors()) {
-                override fun saveCallResult(shareParserResult: ShareParserResult) {
+                override fun saveCallResult(item: ShareParserResult) {
                     saved.set(true)
                 }
 
+                override fun shouldFetch(data: List<OCShare>?): Boolean {
+                    return true;
+                }
+
                 override fun loadFromDb(): LiveData<List<OCShare>> {
                     dbData.value = fetchedDbValue
                     return dbData
```

Comment:
Could we use one line function using apply?

`override fun loadFromDb(): LiveData<List<OCShare>> = dbData.apply{ value = fetchedDbValue }`